packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js

```diff
@@ -77,6 +77,59 @@ export default class MeetingInfoV2 {
     });
   }
 
+  /**
+     * Fetches meeting info from the server
+     * @param {String} conversationUrl one of many different types of destinations to look up info for
```

Comment:
this comment doesn't seem right - you can only pass the conversation url, no other destination will work with this function, right?

Reply:
done